contracts/HATVault.sol

```diff
@@ -272,7 +272,7 @@ contract HATVault is IHATVault, ERC4626Upgradeable, OwnableUpgradeable, Reentran
                 revert ChallengedClaimCanOnlyBeApprovedByArbitratorUntilChallengeTimeoutPeriod();
             }
             // the arbitrator can update the bounty if needed
-            if(claim.arbitratorCanChangeBounty) {
+            if(claim.arbitratorCanChangeBounty && _bountyPercentage != 0) {
```

Comment:
Are you adding this bc the arbitrator should be be able to call approveClaim but set the bounty to 0 (effectively dismissing the claim). I kind of agree that this is a thing, but this check still allows for a bountypercentage of 0.1%, which is pretty much the same thing. 
I am slightly for not adding this check for that reason

Reply:
No, the point is to allow the arbitrator to call approve but not change the percentage, it's convenient option since arbitrator could just specify the percentage actually specified, but this just saves mistakes or in the arbitrator contract case, it saves the need to query the contract to know the percentage